docs/example-scenario/data/data-warehouse.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Example scenario for data warehousing and analytics in Azure
+description: Use a data warehouse in Azure to consolidate data from multiple sources and optimize data analytics.
+author: alexbuckgit
+ms.date: 08/16/2018
+---
+
+# Example scenario for data warehousing and analytics in Azure
+
+Build a data pipeline to integrate large amounts of data from multiple sources into a cloud-scale analytics platform. This scenario is relevant for:
+* Sales and marketing
+* E-commerce and retail
+* Healthcare
+* Other solutions requiring advanced analysis of large datasets
+
+## The scenario
+
+A sales and marketing company builds incentive programs. These programs reward customers, suppliers, salespeople, and employees. Data is fundamental to these programs, and the company wants to improve the insights gained through data analytics using Azure.
```

Comment:
In this example, we're demonstrating a sales and marketing company that builds incentive programs.  These programs...